srcpkgs/screenkey/template

```diff
@@ -1,15 +1,16 @@
 # Template file for 'screenkey'
 pkgname=screenkey
-version=1.4
-revision=2
+version=1.5
+revision=1
 build_style=python3-module
 hostmakedepends="intltool python3-setuptools python3-distutils-extra
  python3-Babel"
-depends="gtk+3 libX11 python3-gobject"
-checkdepends="${depends} libXtst"
+depends="gtk+3 libX11 python3-gobject python3-cairo python3-dbus"
 short_desc="Screencast tool to display your keys"
 maintainer="Steve Prybylski <sa.prybylx@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="http://www.thregr.org/~wavexx/software/screenkey"
+changelog="https://www.thregr.org/~wavexx/software/screenkey/NEWS.html"
 distfiles="${homepage}/releases/screenkey-${version}.tar.gz"
-checksum=d7347e31fab4a5b18da431e3fc26f39700e99a505eb0c0168295adc12c8dd4f7
+checksum=cc8471b83f7ba7a754e1da0631cfa9c32b9217da93597afc1c3283e3a1ae4112
+make_check=no # has no tests
```

Comment:
I don't think this matters if it at least checks dependencies.

Reply:
I force pushed now without disabling checks so maybe you can help me with that.

Adding `python3-pip` and `python3-wheel` to checkdepends doesn't help as it just tries to build dbus-python and fails.

When I instead add `python3-pytest` to $checkdepends it *errors* with
```
=> screenkey-1.5_1: running do_check ...
============================= test session starts ==============================
platform linux -- Python 3.10.3, pytest-6.2.5, py-1.10.0, pluggy-0.13.1
rootdir: /builddir/screenkey-1.5
collected 0 items

============================ no tests ran in 0.02s =============================
```

That's the reason I disabled checks.

Reply:
Adding python3-pip and python3-wheel defeats the purpose of checks because it creates its own environment with its own versions that don't match void versions.

I see that now the check fails with 
```
error: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpj7e50gbs', '--quiet', 'dbus-python']' returned non-zero exit status 1.
```
which usually means the dbus-python module is missing, but as far as I can tell, dbus-python is our python3-dbus. I am not really sure how the detection is done, but our python3-dbus is not detected.

Reply:
I think the command `/usr/bin/python3 -m pip ...` fails because pip is not installed :/

Reply:
I think our `python3-dbus` is incomplete. It seems to lack a `dbus_python-1.2.18-py3.10.egg-info`, which is present when I use `pip` to install `dbus-Python` in a virtual environment. The test suite probably fails to recognize the package without the egg metadata present.

Reply:
Please check if https://github.com/void-linux/void-packages/pull/36262 resolves this issue.

Reply:
Yes it does :) thank you!

Reply:
I rebased on master